spec/producer_spec.rb

```diff
@@ -296,6 +296,16 @@ def self.partition_key(payload)
       )
     end
 
+    it 'should error if no topic is given' do
+      expect { MyProducer.publish_list(
+          [{ 'test_id' => 'foo', 'some_int' => 123 },
+           { 'test_id' => 'bar', 'some_int' => 124 }],
+          topic: ""
+      ) }.to raise_error(RuntimeError,
+                             'Topic not given! Please specify the topic.')
+
+    end
```

Comment:
Should also test the case where you create a producer with no topic configured and don't pass any topic in explicitly.

Reply:
Added. @dorner

Reply:
oh I forgot to fix the linter issues, gonna do that after my meeting